convert.py

```diff
@@ -39,21 +40,31 @@ def convert(flags):
         groups.append(group)
 
     manifest = generate_manifest(flags.survey_title, groups)
-    with open(flags.manifest_out, 'w') as f:
+    manifest_file = os.path.join(flags.manifest_out, flags.survey_title + '.yaml')
+    with open(manifest_file, 'w') as f:
         yaml.dump(manifest, f, default_flow_style=False)
 
 
-def create_yaml_blocks(flags, block):
-    try:
-        with open('{}/{}.yaml'.format(flags.blocks_out, block['id']), 'r+') as f:
+def create_yaml_block(flags, block):
+    """
+    Checks whether block name exists, and if content matches that of an existing block
+    :param flags: Parses user input to define block file names
+    :param block: Generated manifest block
+    :return: Returns Block Yaml files and prevents creation of existing blocks
+    """
+    block_file = os.path.join(flags.blocks_out, block['id'] + '.yaml')
+    block_file_variant = os.path.join(flags.blocks_out, block['id'] + '-' + flags.survey_variant + '.yaml')
```

Comment:
this should be defined before line 63 as it's only needed if going down that route